### DISCUSSION TITLE
Fix to .zip file downloaded instead of .epub

### DISCUSSION THREAD

**Discussion by lockone00** [MakeYourLifeEasier/novel-ebook.com#138] [General]
![Screenshot_2021-03-12-02-13-22-930_com mi android globalFileexplorer](https://user-images.githubusercontent.com/80481539/110836411-f633bf80-82da-11eb-9eb7-450c5b53fb2c.jpg)
![Screenshot_2021-03-12-02-25-53-019_com mi android globalFileexplorer](https://user-images.githubusercontent.com/80481539/110836428-ffbd2780-82da-11eb-9f50-a6a1931a742a.jpg) 
Just go to file manager and rename then delete the .zip extension so the .epub extension will be left and that should fix it. Hope this helps. :)

**Comment by MichaelGuardian** [comment]
This is actually an android bug, nothing I can do. Epub files are essentially zip files so for some reason, some versions of android add the extension automatically. I double checked the code to be sure but the file extension is declared as .epub.
